OsmAnd-java/src/main/java/net/osmand/router/RouteResultPreparation.java

```diff
@@ -1808,21 +1816,13 @@ protected TurnType createSimpleKeepLeftRightTurn(boolean leftSide, RouteSegmentR
 
 
 	private int[] createCombinedTurnTypeForSingleLane(RoadSplitStructure rs, double currentDeviation) {
-		rs.attachedAngles.add(currentDeviation);
-		Collections.sort(rs.attachedAngles, new Comparator<Double>() {
-			@Override
-			public int compare(Double c1, Double c2) {
-				return Double.compare(c1, c2);
-			}
-		});
-
 		int size = rs.attachedAngles.size();
 		boolean allStraight = rs.allAreStraight();
 		int[] lanes = new int[1];
 		int extraLanes = 0;
 		double prevAngle = Double.NaN;
 		// iterate from left to right turns
-		for (int i = size - 1; i >= 0; i--) {
```

Comment:
What the difference? Situation should be symmetric cause we have the right driving countries and left driving driving countries, fix couldn't rely on changing iteration order